docs/operator/policy_packages.md

```diff
@@ -28,6 +28,8 @@ VO. In this case, the configuration parameter or environment variable name
 is suffixed with the VO name (for example, `package-vo1` or
 `RUCIO_POLICY_PACKAGE_VO1`).
 
+[The API for policy packages can be found here.](pathname:///html/rse_policies/rsemanager.html)
```

Comment:
Not sure if it could be confusing to call this the API for policy packages - most functions listed here https://rucio.github.io/documentation/html/rse_policies/rsemanager.html seem to be unrelated to policy packages